server/database/schemas/Comment.js

```diff
@@ -0,0 +1,42 @@
+const mongoose=require("mongoose")
+
+const commentSchema=new mongoose.Schema({
+    posted_by: {
+        type:mongoose.SchemaTypes.ObjectId,
+        required:true,
+    },
+    replied_by: {
+        type:[mongoose.SchemaTypes.ObjectId],
+        required:true
+    },
+    message:{
+     type:String,
+     required:true
+    },
+    flagged:{
+    type:Boolean,
+    required:true
+    } ,
+    replies: {
+    type:Number,
+    required:true}
+    ,
+    votes: {
+    type:Number,
+    required:true }
+
+})
+
+module.exports=mongoose.model("Comment",commentSchema)
+
+/*
```

Comment:
Please delete this comment. It's fine to have it for reference while developing but we don't need it in production.

Reply:
The comment starting at line 32 I mean*